gips/data/landsat/landsat.py

```diff
@@ -864,18 +865,15 @@ def process(self, products=None, overwrite=False, **kwargs):
                     cfmask[cfmask == 2] = 0
 
                     verbose_out("writing " + fname, 2)
-                    imgout = gippy.GeoImage(fname, img, gippy.GDT_Byte, 1)
-                    imgout.SetBandName('Land mask', 1)
-                    imgout[0].Write(cfmask)
+                    imgout = gippy.GeoImage(img, fname, 1, 'byte')
```

Comment:
Oops; I think I forgot the `create_from`; this needs to be `imgout = gippy.GeoImage.create_from(img, fname, 1, 'byte')`